roles/openshift_health_checker/library/search_journalctl.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+"""Interface to journalctl"""
+
+from time import time
+import json
+import re
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+class InvalidMatcherRegexp(Exception):
+    """Exception class for invalid matcher regexp."""
+    pass
+
+
+class InvalidLogEntry(Exception):
+    """Exception class for invalid / non-json log entries."""
+    pass
+
+
+class LogInputSubprocessError(Exception):
+    """Exception class for errors that occur while executing a subprocess"""
```

Comment:
For consistency sake, please end the docstrings with a period. This one is different than the similar classes above.
The module docstring in this file and the one below are also missing punctuation.